laravel-app/resources/views/home.blade.php

```diff
@@ -33,6 +33,15 @@
             </div>
         </div>
     </div>
+    
+    {{-- Add a button that directs you to another page --}}
```

Comment:
why needed. a comment here?

Reply:
for reference sir that I added a button, it will be removed sir in the future when improving the frontend sir.